modules/kuberay-operator/kuberay.tf

```diff
@@ -32,8 +32,6 @@ module "kuberay-workload-identity" {
   project_id          = var.project_id
   roles               = ["roles/cloudsql.client", "roles/monitoring.viewer"]
 
-  automount_service_account_token = true
```

Comment:
FYI this broke the RAG application. See: https://github.com/GoogleCloudPlatform/ai-on-gke/commit/6b844bcfa3ead58a10ded54c215e61e856c8ffae#r139575280